src/main/java/uk/gov/justice/hmpps/prison/api/resource/PrisonRollCountResource.kt

```diff
@@ -0,0 +1,108 @@
+package uk.gov.justice.hmpps.prison.api.resource
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.tags.Tag
+import org.springframework.http.HttpStatus
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.hmpps.prison.api.model.ErrorResponse
+import uk.gov.justice.hmpps.prison.core.SlowReportQuery
+import uk.gov.justice.hmpps.prison.security.VerifyAgencyAccess
+import uk.gov.justice.hmpps.prison.service.PrisonRollCount
+import uk.gov.justice.hmpps.prison.service.PrisonRollCountService
+
+@RestController
+@Tag(
+  name = "Establishment Roll Count",
```

Comment:
not a great fan of this tag.  Most tags at present are single word - wondering if it could be included in the existing prison tag instead - https://prison-api-dev.prison.service.justice.gov.uk/swagger-ui/index.html#/prison